config.yaml

```diff
@@ -0,0 +1,6 @@
+operators:
```

Comment:
does this `config.yaml` have to be at `src`, are we using this somewhere?

Reply:
The config.yaml needs to be at the directory where we run our script since we are passing it to Feluda class instance.